pkg/network/ovn_kubernetes.go

```diff
@@ -225,7 +238,14 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 		data.Data["IsSNO"] = false
 	}
 
-	manifests, err := render.RenderDir(filepath.Join(manifestDir, "network/ovn-kubernetes"), &data)
+	var manifestPath string
+	if hypershift != "true" {
+		manifestPath = "network/ovn-kubernetes"
+	} else {
+		manifestPath = "hypershift/ovn-kubernetes"
+	}
+
+	manifests, err := render.RenderDir(filepath.Join(manifestDir, manifestPath), &data)
```

Comment:
Idea: what if there are 3 directories under `ovn-kubernetes`: `common`, `self-hosted`, and `managed`? Then, we always render `common` along with whichever directory the mode calls for. @kyrtapz

Reply:
Good point re: the ordering. Sure, we can do that. I'll file a PR that you can cherry-pick.